core/classes/trojsten-tasks.cls

```diff
@@ -0,0 +1,16 @@
+\NeedsTeXFormat{LaTeX2e}[2010/01/01]
+\ProvidesClass{trojsten-tasks}[2015/09/08 Trojsten tasks booklet]
+
+\RequirePackage{pgfkeys, pgfopts}
+
+\pgfkeys{
+    /TROJSTEN/.cd,
+    TROJSTEN/.code = \wlog{Value '#1' given},
+    TROJSTEN/.store in = \TROJSTEN@store
+}
+
+\DeclareOption{fks}{\loadSeminar{fks}}
+\DeclareOption{kms}{\loadSeminar{kms}}
+\DeclareOption{ksp}{\loadSeminar{ksp}}
+\DeclareOption{ufo}{\loadSeminar{ufo}}
+\DeclareOption{prask}{\loadSeminar{prask}}
```

Comment:
Štruktúra projektu: momentálne je folder `core/` ktorý obsahuje subfoldre napríklad `config/`, `iconography/`, … ktoré in turn obsahujú subfoldre jednotlivých seminárov / súťaží. Bolo by možné / ako by bolo ťažké prerobiť túto štruktúru následovne:

  - Folder `core/` obsahuje iba jadro DGS. Niesu tam žiadne veci špecifické pre nejaký seminár.
  - Vedľa foldra `core/` sa zjaví folder `modules/` ktorý obsahuje subfoldre modulov (`ksp/`, `fks/`, …) a tieto moduly obsahujú všetky veci pre danú súťaž (ikonografiu, templaty, ľubovolný python súbor extendujúci classu modul overridujúci defaultné správanie, …).
  - Core nevie nič o konkrétnych moduloch, iba to, že user keď buildi tak si vie do configu buildu napísať `module: fks` a zbuildí sa to pre FKS.
  - Moduly nevedia vzájomne o sebe, sú úplne izolované.

Výhoda je jasná separácia, ak sa pridáva súťaž je jasné čo treba spraviť (spísať modul na jedno miesto as opposed to toto hodiť do tohto foldra, do týchto 5tich súborov niečo dopísať a celé špagety ešte viac previazať).

Je niečo také feasible?